ArjunaJTA/cdi/pom.xml

```diff
@@ -298,5 +298,19 @@
         <jvm.args.modular>--add-modules=java.se</jvm.args.modular>
       </properties>
     </profile>
+    <profile>
+      <id>jdk17</id>
+      <activation>
+        <jdk>[17,)</jdk>
+      </activation>
+      <properties>
+        <jvm.args.modular>--add-modules=java.se
```

Comment:
I am not sure if this java.se module required to be added here because I guess https://github.com/jbosstm/narayana/blob/master/ArjunaJTA/cdi/pom.xml#L298 might already take effect?

Reply:
Okay Tom, sure I will remove it from JDK17 profile